ui/src/components/imquery/ConceptSelect.vue

```diff
@@ -20,6 +20,10 @@
         <div class="value-list-item" v-for="[index, value] of values.entries()">
           <EntailmentOptionsSelect :entailment-object="value" />
           <AutocompleteSearchBar
+            :quick-type-filters-allowed="[IM.CONCEPT, IM.CONCEPT_SET]"
+            :default-quick-type-filter="isObjectHasKeys(value, ['memberOf']) ? IM.CONCEPT_SET : IM.CONCEPT"
```

Comment:
shouldn't need to set both quick-type-filters-allowed and default-quick-type-filters

Shouldn't be able to set default filters at all actually.